src/helpers/buildEnabledRouteType.ts

```diff
@@ -0,0 +1,19 @@
+import * as t from 'io-ts';
```

Comment:
this type error is because you are not doing the type override:

```
/** Type override */
export type OpenAIRouteName = t.TypeOf<typeof OpenAIRouteName>;
```

Reply:
Fixed